CHANGES.rst

```diff
@@ -1,4 +1,4 @@
-﻿3.0 (unreleased)
+﻿3.0.0 (unreleased)
 =================
```

Comment:
Yes, it should stay 3.0 https://github.com/astropy/astropy/pull/6375#discussion_r128350546

The bot should actually be able to recognize it (and it does for e.g. #6373), I guess the added whitespace messes things up here.